db/migrate/20150922185348_add_admin_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdminToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :admin, :boolean, default: false
```

Comment:
Nice work adding a default--booleans should alllllways have defaults otherwise bizarre things happen :)